SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -661,8 +661,11 @@ Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(con
 		}
 	}
 
-	// Don't add NCD component to player controller and server only actors as we don't want client's to gain interest in them
-	if (GetDefault<USpatialGDKSettings>()->bEnableNetCullDistanceInterest && !Actor->IsA<APlayerController>()
+	checkf(!Actor->IsA<APlayerController>() || Actor->bOnlyRelevantToOwner,
+		   TEXT("Player controllers must be have bOnlyRelevantToOwner enabled."));
+	// Don't add NCD component to actor's only relevant to their owner (player controllers etc.) and server only actors
+	// as we don't want client's to otherwise gain interest in them.
```

Comment:
```suggestion
	// as we don't want clients to otherwise gain interest in them.
```
Sorry for only commenting on text